packages/ng/pagination/pagination.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, effect, input, output } from '@angular/core';
+import { getIntl, IntlParamsPipe } from '../core/translate';
+import { LU_PAGINATION_TRANSLATIONS } from './pagination.translate';
+
+@Component({
+	selector: 'lu-pagination',
+	templateUrl: './pagination.component.html',
+	imports: [IntlParamsPipe],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaginationComponent {
+	protected intl = getIntl(LU_PAGINATION_TRANSLATIONS);
+
+	readonly isFirstPage = input.required<boolean>();
+	readonly isLastPage = input.required<boolean>();
+
+	readonly from = input<number>();
+	readonly to = input<number>();
+	readonly itemsCount = input<number>();
```

Comment:
`input` signals should either be required or have a default value, here it's left to implicit `undefined` which can quickly become annoying to handle, or make them `| null` explicitly.